src/lavinmq/amqp/client.cr

```diff
@@ -97,15 +97,15 @@ module LavinMQ
       end
 
       private def read_loop
-        i = 0
+        received_bytes = 0_i32
```

Comment:
definitely unsigned, but maybe even u64
```suggestion
        received_bytes = 0_u32
```

Reply:
I think u32 should be enough? would mean we can handle message sizes up to 4gb